src/graph/graph.user.ts

```diff
@@ -15,11 +15,16 @@ import { prepScopes } from '../utils/GraphHelpers';
  * @returns {(Promise<User>)}
  * @memberof Graph
  */
-export function getMe(graph: IGraph): Promise<User> {
-  return graph
-    .api('me')
-    .middlewareOptions(prepScopes('user.read'))
-    .get();
+export async function getMe(graph: IGraph): Promise<User> {
+  const batch = graph.createBatch();
+  batch.get('user', 'me', ['user.read']);
```

Comment:
Batch seems unnecessary here. We're only making one get request.

Reply:
For context originally mgt-person makes a batch call to get 'me' so when I refactored that code out I've updated it here. Can def keep this as single request if we agree to the former question ☝🏼to create a batch to get both person and person photo. I am ok with either implementation.

Reply:
Seeing how `mgt-person` and `mgt-person-card` do pretty much the same data calls, you may find we don't need this at all. It's all about whatever works best for the use case.

Reply:
done